test/e2e/operator_test.go

```diff
@@ -143,6 +144,21 @@ func TestMain(m *testing.M) {
 		os.Exit(1)
 	}
 
+	err = ensureOperandNamespace()
+	if err != nil {
+		fmt.Printf("Failed to create external-dns namespace: %v\n", err)
+	}
+
+	err = ensureOperandRole()
+	if err != nil {
+		fmt.Printf("Failed to create role external-dns-operator in ns external-dns: %v\n", err)
```

Comment:
Don't we want to fail the test? What's the point of continuing?

Reply:
Basically mostly this fails when the role already exists,  and we just want to go on...
Otherwise, we will need to take care of cleanups, we can also do that

Reply:
`ensure*` functions can do a `Get` call before `Create` to check if it's already there and return `error` only in case of a real failure (creation).